designer/server/src/test/scala/pl/touk/nussknacker/test/utils/scalas/FutureExtensions.scala

```diff
@@ -0,0 +1,15 @@
+package pl.touk.nussknacker.test.utils.scalas
+
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
+
+trait FutureExtensions {
```

Comment:
We already have a solution for that - `PatientScalaFutures` which is just a little bit tunned scalatest's `ScalaFutures`